CONTRIBUTING.md

```diff
@@ -11,5 +11,7 @@ Please note we have a code of conduct, please follow it in all your interactions
    build.
 2. Update the README.md with details of changes to the interface, this includes new environment
    variables, exposed ports, useful file locations and container parameters.
+3. Make sure that your code follows the formatting rules enforced within this repository. This can be checked by running:
+`flake8 src && isort -c src && black --check src`, assuming that you have required tools installed from `requirements-dev.txt` (note the `axblack` for of `black` used).
```

Comment:
Just a proposition, I had to do some cleanup after merging `master` so maybe this can be enforced at this level here?

Reply:
That's fine for now. As you can see in #45, the plan for the future is to use [pre-commmit](https://pre-commit.com).
So this will probably need to be fixed then, but I wouldn't worry about it just yet. It's fine for now.

Reply:
This is now updated to include `mypy` but exclude `isort` and `black` as they are out of scope.